src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,162 @@
+package parser;
+
+import tasklist.*;
+import exception.*;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * The Parser class is responsible for handling input commands.
+ */
+public class Parser {
+
+    /**
+     * Handles commands regarding tasks(ie. todos, deadlines, events).
+     * @param input The command input.
+     * @param newList The current TaskList.
+     * @throws DukeException if there is an error with the input or input is not given in the correct format.
+     */
+    public static void commandTask(String input, TaskList newList) throws DukeException {
+
+        String[] splitString = input.split(" ",2);
+
+        if (splitString[0].equals("todo")) {
+            if (splitString.length == 1) {
+                throw new DukeException("    ☹ OOPS!!! The description of a todo cannot be empty.");
+            } else {
+                String info = splitString[1];
+                Todos newTodo = new Todos(info);
```

Comment:
I believe Class name should be singular, so you might want to rename Todos to Todo!